apps/bridge-dapp/src/containers/DepositContainer/DepositContainer.tsx

```diff
@@ -72,6 +72,10 @@ export const DepositContainer = forwardRef<
       MainComponentVariants | undefined
     >(undefined);
 
+    const setMainComponentHandler = (component: string | undefined) => {
+      setMainComponentName(component as MainComponentVariants);
+    };
```

Comment:
It's a bit confusing here. Why did you define the `component: string | undefined` and then cast the variable to type `MainComponentVariants`?

I saw that you use this function to reset the `mainComponentName` to `undefined` so consider using something like this.

```jsx

const resetMainComponent = useCallback(() => {
	setMainComponentName(undefined);
}, [setMainComponentName]);

```